### DISCUSSION TITLE
Aggiornamento 1.5.0.

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#318] [Richieste di aiuto] [answered]
Salve, verso che giorno sarà rilasciato l'aggiornamento? Il rientro a scuola si avvicina e dovremmo effettuare il passaggio all'anno successivo, l’inserimento delle classi e l'assegnazione delle cattedre😅

**Comment by trinko** [comment, marked as answer]
Se tutto va bene, tra oggi e domani.

**Comment by sempliciniluigi89** [comment]
> Salve, verso che giorno sarà rilasciato l'aggiornamento? Il rientro a scuola si avvicina e fare il passaggio all'anno successivo, l'inserimento delle classi e l'assegnazione delle cattedre😅

E' vero,siamo quasi agli sgoccioli  delle vacanze estive e servirebbe che il registro fosse aggiornato e funzionante in ogni sua parte
